### ISSUE TITLE
ejabberdctl commands appear hanged and use 100% of one core

### ISSUE THREAD

**Issue by licaon-kter** [processone/ejabberd#2939]
...running HEAD (f0bbc05b7a5f5bae003efc5c03679e06040dfec7) on arm64, nothing happens, you need to hit CTRL-C twice to stop it. Eg. `status` or `reload_config`, stopping the server takes about 10 mins (from 10s max), during this time at least one core is at 100%

**git bisect** says:
```
696c64064bd643710e56e7c88b70a41dfb15cd8a is the first bad commit
commit 696c64064bd643710e56e7c88b70a41dfb15cd8a
Author: Evgeny Khramtsov <ekhramtsov@process-one.net>
Date:   Fri Jul 12 13:58:46 2019 +0300

    Add call/5 and multicall/5 to ejabberd_cluster

:040000 040000 0dc7e80925960a85ae57e70732a88f6fa1da47af af6cb87e7fa4c802832d211984f099c7d67cf4c6 M      src
```

No errors in logs.

Note that in https://github.com/processone/ejabberd/commit/53dacb382590ec9fc350c18754c04aa822bbaadf `ejabberdctl` commands failed with
```
{"init terminating in do_boot",{undef,[{ejabberd_cluster,call,['ejabberd@localhost',ejabberd_ctl,process,[["status"]],60000],[]},{ejabberd_ctl,start,0,[{file,"src/ejabberd_ctl.erl"},{line,95}]},{init,start_em,1,[]},{init,do_boot,3,[]}]}}
init terminating in do_boot ({undef,[{ejabberd_cluster,call,[ejabberd@localhost,ejabberd_ctl,process,[_],60000],[]},{ejabberd_ctl,start,0,[{_},{_}]},{init,start_em,1,[]},{init,do_boot,3,[]}]})

Crash dump is being written to: ...
```
...and that got fixed in 696c64064bd643710e56e7c88b70a41dfb15cd8a to no longer show an error, but now it doesn't show anything at all.

Running a command on HEAD (f0bbc05b7a5f5bae003efc5c03679e06040dfec7), 1 core at 100%, **etop -sort reductions** says:
```
Erlang/OTP 21 [erts-10.2.4] [source] [64-bit] [smp:4:4] [ds:4:4:10] [async-threads:1]

Eshell V10.2.4  (abort with ^G)
(etop@rpi3)1> 

========================================================================================
 etop@rpi3                                                                 00:32:05
 Load:  cpu         0               Memory:  total       15596    binary        581
        procs      47                        processes    4554    code         4770
        runq        0                        atom          250    ets           355

Pid            Name or Initial Func    Time    Reds  Memory    MsgQ Current Function
----------------------------------------------------------------------------------------
<0.69.0>       user                     '-'   12338   42288       0 group:server_loop/3 
<0.83.0>       etop_txt:init/1          '-'    5621   42260       0 etop:update/1       
<0.68.0>       user_drv                 '-'    1151   13732       0 user_drv:server_loop
<0.45.0>       application_master:i     '-'       2    3920       0 application_master:m
<0.67.0>       supervisor_bridge:us     '-'       2    2776       0 gen_server:loop/7   
<0.71.0>       kernel_config:init/1     '-'       2    2732       0 gen_server:loop/7   
<0.0.0>        init                     '-'       1   10720       0 init:boot_loop/2    
<0.1.0>        erts_code_purger         '-'       1    2600       0 erts_code_purger:wai
<0.2.0>        erts_literal_area_co     '-'       1    2600       0 erts_literal_area_co
<0.3.0>        erts_dirty_process_s     '-'       1    2600       0 erts_dirty_process_s
========================================================================================
```

**Comment by zinid** [comment]
Fixed in 6013b4a3a6d80c5542bc55a4c4360d07fae256f5

**Comment by licaon-kter** [comment]
Confirmed fixed in HEAD
